guides/source/action_mailbox_basics.md

```diff
@@ -18,30 +18,32 @@ After reading this guide, you will know:
 What is Action Mailbox?
 -----------------------
 
-Action Mailbox routes incoming emails to controller-like mailboxes for
-processing in Rails. It ships with ingresses for Mailgun, Mandrill, Postmark,
-and SendGrid. You can also handle inbound mails directly via the built-in Exim,
-Postfix, and Qmail ingresses.
+Action Mailbox routes incoming emails to controller-like mailboxes for processing in your Rails application. Action Mailbox is for receiving inbound email, while [Action Mailer](https://guides.rubyonrails.org/action_mailer_basics.html) is for *sending* emails from your Rails application.
 
-The inbound emails are turned into `InboundEmail` records using Active Record
-and feature lifecycle tracking, storage of the original email on cloud storage
-via Active Storage, and responsible data handling with
-on-by-default incineration.
+The inbound emails are routed asynchronously using Active Job to one or several dedicated mailboxes. These emails are turned into `InboundEmail` records using Active Record, which are capable of interacting directly with the rest of your domain model.
 
-These inbound emails are routed asynchronously using Active Job to one or
-several dedicated mailboxes, which are capable of interacting directly
-with the rest of your domain model.
+`InboundEmail` records also provide lifecycle tracking, storage of the original email via Active Storage, and responsible data handling with on-by-default incineration.
+
+Action Mailbox ships with ingresses for external email providers such as Mailgun, Mandrill, Postmark, and SendGrid. You can also handle inbound mails directly via the built-in Exim, Postfix, and Qmail ingresses.
 
 ## Setup
 
-Install migrations needed for `InboundEmail` and ensure Active Storage is set up:
+Action Mailbox has a few moving parts. First you'll run the Action Mailbox installer. Then configure a chosen ingress for handling incoming email. You're then ready to add Action Mailbox routing, create mailboxes and start processing incoming emails.
 
-```bash
-$ bin/rails action_mailbox:install
-$ bin/rails db:migrate
-```
+To start let's install Action Mailbox:
+
+1. `bin/rails action_mailbox:install` - this will create an `application_mailbox.rb` file and copy over migrations.
+2. `bin/rails db:migrate` - this will run the Action Mailbox and Active Storage migrations.
```

Comment:
One thing that just occurred to me, related to what Amanda mentioned above: we might want to try to keep commands split into their own blocks, that makes it easier to copy manually, but also gives readers a free "copy" button to click:

<img width="653" alt="Screenshot 2024-02-05 at 11 59 38" src="https://github.com/bhumi1102/rails/assets/26328/a1dd88f0-c73f-4cf5-9ae2-2bca6554f3b6">

We could use comments to connect things: (they aren't copied with that button)


````suggestion
```bash
# Creates an `application_mailbox.rb` file, copies over migrations.
$ bin/rails action_mailbox:install
```

```bash
# Runs the Action Mailbox and Active Storage migrations.
$ bin/rails db:migrate
```
````

The output would look something like this:

<img width="663" alt="Screenshot 2024-02-05 at 12 04 52" src="https://github.com/bhumi1102/rails/assets/26328/092c8d23-55fd-42a7-a405-f59e30153e61">

Or with paragraphs:

````suggestion
```bash
$ bin/rails action_mailbox:install
```

This will create an `application_mailbox.rb` file and copy over migrations.

```bash
$ bin/rails db:migrate
```

This will run the Action Mailbox and Active Storage migrations.
````

_(note that I merged the last sentence with the subsequent paragraph in the screenshot below)_

<img width="686" alt="Screenshot 2024-02-05 at 12 06 30" src="https://github.com/bhumi1102/rails/assets/26328/5600e636-5375-459f-9c27-e9412eace8ba">

Reply:
Good point about the 'copy' button! I like the one with paragraphs a bit better, so going with that suggestion.